pyquil/tests/test_gates.py

```diff
@@ -137,6 +139,27 @@ def test_forked_gate():
     assert g.out() == "FORKED RX(0,1.0) 1 0"
     g = RX(0.0, 0).forked(1, [1.0]).forked(2, [2.0, 3.0])
     assert g.out() == "FORKED FORKED RX(0,1.0,2.0,3.0) 2 1 0"
+    g = RX(0.0, 0).forked([1], [1.0])
+    assert g.out() == "FORKED RX(0,1.0) 1 0"
+    g = RX(0.0, 0).forked([1, 2], [1.0, 2.0, 3.0])
```

Comment:
open to suggestions about whether `alt_params` should be a flat `Sequence` or nested `Sequence`s when passing more than one `fork_qubit`, i.e.

```python
RX(0.0, 0).forked([1, 2], [[1.0], [2.0, 3.0]])
```

rather than

```python
RX(0.0, 0).forked([1, 2], [1.0, 2.0, 3.0])
```